var.tfvars

```diff
@@ -18,11 +18,13 @@ private_key                 = ""
 public_key                  = ""
 rhel_subscription_username  = ""
 rhel_subscription_password  = ""
+#rhcos_kernel_options        = ["slub_max_order=0"]
```

Comment:
Let's not set any defaults and make this optional

Reply:
This one is commented.. that implies it is optional? :-)

Reply:
I was not clear.. why do we have to mention the kernel params and keep it commented ? Instead we keep it uncommented and set to empty list or we keep commented but empty list.

Reply:
Commented because the default is always [] as set in the variables.tf. Whoever wants to use it just uncomment it and start using it the way specified as an example. Served as multiple purpose.

Ok, I will make it uncommected and set to the default [] here.